corehq/apps/reports/standard/users/reports.py

```diff
@@ -237,11 +262,27 @@ def _user_history_details_cell(self, changes, domain):
                 primary_changes[properties[key]] = value
                 all_changes[properties[key]] = value
         more_count = len(all_changes) - len(primary_changes)
-        return render_to_string("reports/standard/partials/user_history_changes.html", {
-            "primary_changes": self._html_list(primary_changes),
-            "all_changes": self._html_list(all_changes),
-            "more_count": more_count,
-        })
+
+        if for_export:
+            def _convert_dict_to_string(changes_dict):
+                s = ''
+                for i, key in enumerate(changes_dict):
+                    if not key:
+                        continue
+                    elif not changes_dict[key]:
+                        s += key + ": None"
+                    else:
+                        s += key + ': ' + str(changes_dict[key])
```

Comment:
Stylistic comment: do you like f-strings? I think `f": {changes_dict[key]}"` is more readable than explicitly casting to a string.

Reply:
I love f-strings. Idk about it here but I don't feel strongly so I changed it